applications/nrf_desktop/src/modules/hid_state.c

```diff
@@ -1053,6 +946,35 @@ static void report_issued(const void *subscriber_id, uint8_t report_id, bool err
 	}
 }
 
+static struct report_state *find_next_report_state(struct report_data *rd)
+{
+	struct report_state *rs = NULL;
+	int8_t sub_prio = SUB_PRIO_LOWEST;
+
+	for (size_t i = 0; i < ARRAY_SIZE(state.subscriber); i++) {
+		if (!state.subscriber[i].id) {
+			continue;
+		}
+
+		struct subscriber *sub = &state.subscriber[i];
+
+		for (size_t j = 0; j < ARRAY_SIZE(sub->state); j++) {
+			struct report_state *rs_temp = &sub->state[j];
```

Comment:
s/temp/other or something - this is nothing temporary

Reply:
changed